pkg/controller/controller_instance_test.go

```diff
@@ -675,8 +675,10 @@ func TestReconcileServiceInstanceWithParameters(t *testing.T) {
 				InstanceID:        testServiceInstanceGUID,
 				ServiceID:         testClusterServiceClassGUID,
 				PlanID:            testClusterServicePlanGUID,
-				Context:           testContext,
-				Parameters:        tc.expectedParams,
+				OrganizationGUID:  testClusterID,
+				// TODO SpaceGUID:         testNamespaceGUID,
```

Comment:
What's up with the lingering TODO? If it needs to stick around, let's provide more info in the comment on what it's waiting for.

Farther down it's being set so I'm not sure why these can't be set too, but I really don't know what SpaceGUID is. 😀

Reply:
@MHBauer any reason we're not just getting it to the Kube namespace guid?  In CF, "Space" is similar to a Kube "namespace" - just a grouping mechanism for apps. Typically, people have a space for dev, test and prod.

Reply:
It doesn't look like there's any interesting setup required to make SpaceGUID work?

If we removed the comments and started setting `SpaceGUID:         testNamespaceGUID` here just like we already do further below, it seems like it would just work (I haven't tried that). Since we are kind of touching this code, if it doesn't cause test failures, I'd prefer to fix this lingering todo while we are in here, or in a follow-up PR.

Reply:
Every test with a TODO fails.